src/pages/LinkedAccounts/LinkedAccounts.tsx

```diff
@@ -139,11 +172,86 @@ const LinkedAccounts = () => {
             }}
           />
         </div>
-        {account.linked ? renderUnlinkButton(account) : renderLinkButton(account)}
+        {account.valid && renderValidationOptions()}
+        {renderButton(account)}
+      </div>
+    );
+  };
+
+  const renderValidationOptions = () => {
+    return (
+      <div className="linked-accounts__validation">
+        <div className="linked-accounts__validation-radar">
```

Comment:
add some small text here explaining that this is their contract and they must confirm it by hitting the 'link account' button underneath.